css-positioning/absolute.css

```diff
@@ -0,0 +1,47 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 20px;
+  left: 230px;
```

Comment:
I would use `right` in this case so you don't have such large `px` values. Makes it a bit easier to read and understand